hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/StringUtils.java

```diff
@@ -149,6 +150,13 @@ public static String createStartupShutdownMessage(VersionInfo versionInfo,
         "  java = " + System.getProperty("java.version"));
   }
 
+  /**
+   * The same as String.format(Locale.ENGLISH, format, objects).
+   */
+  public static String format(final String format, final Object... objects) {
+    return String.format(Locale.ENGLISH, format, objects);
+  }
+
```

Comment:
I didn't found usage of this method in all places.
Could you tell me why we add this part if I miss something ?